base64_encode.go

```diff
@@ -0,0 +1,46 @@
+package gophplib
+
+import (
+	"encoding/base64"
+	"fmt"
+	"reflect"
+)
+
+const INTMAX = 2147483647
```

Comment:
Instead of declaring it here, why not use `math.MaxInt32`(2,147,483,647), which is already defined in the math package?